qiskit/ignis/experiments/calibration/cal_base_analysis.py

```diff
@@ -133,13 +134,14 @@ def plot(self, ax: Optional['Axis'] = None, **kwargs) -> 'Figure':
             figure = plt.figure(figsize=kwargs.get('figsize', (6, 4)))
             ax = figure.add_subplot(111)
 
-        xval_interp = np.linspace(self._result.xvals[0], self._result.xvals[-1], 100)
-        yval_fit = self.fit_function(xval_interp, *self._result.fitval)
+        xval_interp = np.linspace(self.x_values[0], self.x_values[-1], 100)
 
-        ax.plot(xval_interp, yval_fit, '--', color='blue')
-        ax.plot(self._result.xvals, self._result.yvals, 'o', color='blue')
+        for series, yvals in self._series.items():
+            yval_fit = self.fit_function(xval_interp, *self._result[series].fitval)
+            ax.plot(xval_interp, yval_fit, '--', color='k')
```

Comment:
color should be changed for each series otherwise bit difficult to see if there are many lines. you can use cmap with the same index for fit line and actual data.
https://matplotlib.org/3.3.2/tutorials/colors/colormaps.html

Reply:
and you may need a label with series name if series length is more than one.

Reply:
Good point. Fixed. Data and fit will no have the same color but slightly different shades. Different series will have different colors.